### PR TITLE
clean up references of cluster auto pause

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/developer-experience/issues/390.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Review by conceptualshark — COMMENTED** [review]
I should have caught this in the last PR; what is the status of autopause as a feature? It looks like it's in the alpha5 release notes, and available in production. Is the limitation captured in these warnings something we should have removed with 8.5 (or in 8.6)? 

Depending on the answer I think we may be able to make the auto-pause warnings a bit clearer.

**Comment by christinaausley** [comment]
> what is the status of autopause as a feature? It looks like it's in the alpha5 release notes, and available in production. Is the limitation captured in these warnings something we should have removed with 8.5 (or in 8.6)? Depending on the answer I think we may be able to make the auto-pause warnings a bit clearer.

CC @ultraschuppi do you have any comments here?

**Comment by mesellings** [comment]
@christinaausley I can probably assist here, but might not be able to look at today - I will take a look tomorrow and should be able to clear up any confusion etc 👍

File: docs/components/concepts/clusters.md [hunk 1]
```diff
@@ -32,13 +32,18 @@ When your Free Trial plan expires, you are automatically transferred to the Free
 
 ### Auto-pause
 
+:::caution
+**Cluster auto-pause** is not yet available and only applies to non-Enterprise clusters.
+:::
+
```
Comment (mesellings) [inline docs/components/concepts/clusters.md:38]: Shouldn't this caution be deleted as auto-pause is out now?

File: docs/components/concepts/clusters.md [hunk 2]
```diff
@@ -32,13 +32,18 @@ When your Free Trial plan expires, you are automatically transferred to the Free
...
 Free Trial `dev` (or untagged) clusters are automatically paused eight hours after a cluster is created or resumed from a paused state. Auto-pause occurs regardless of cluster usage.
 
 You can resume a paused cluster at any time, which typically takes five to ten minutes to complete. See [resume your cluster](/components/console/manage-clusters/manage-cluster.md#resume-a-cluster).
 
 - Clusters tagged as `test`, `stage`, or `prod` do not auto-pause.
 - Paused clusters are automatically deleted after 30 consecutive paused days. You can change the tag to avoid cluster deletion.
 - No data is lost while a cluster is paused. All execution and configuration is saved, but cluster components such as Zeebe and Operate are temporarily disabled until you resume the cluster.
+- When a cluster is paused, not all functionality is limited. For example, you may still execute BPMN timers and BPMN message catch events.
```
Comment (mesellings) [inline docs/components/concepts/clusters.md:46]:
```suggestion
- Paused clusters still retain some functionality. For example, you can still execute BPMN timers and BPMN message catch events.
```
Rewording suggestion

File: versioned_docs/version-8.5/components/concepts/clusters.md [hunk 3]
```diff
@@ -32,13 +32,18 @@ When your Free Trial plan expires, you are automatically transferred to the Free
 
 ### Auto-pause
 
+:::caution
+**Cluster auto-pause** is not yet available and only applies to non-Enterprise clusters.
+:::
+
```
Comment (mesellings) [inline versioned_docs/version-8.5/components/concepts/clusters.md:38]: See previous comment - remove?

File: versioned_docs/version-8.5/components/concepts/clusters.md [hunk 4]
```diff
@@ -32,13 +32,18 @@ When your Free Trial plan expires, you are automatically transferred to the Free
...
 Free Trial `dev` (or untagged) clusters are automatically paused eight hours after a cluster is created or resumed from a paused state. Auto-pause occurs regardless of cluster usage.
 
 You can resume a paused cluster at any time, which typically takes five to ten minutes to complete. See [resume your cluster](/components/console/manage-clusters/manage-cluster.md#resume-a-cluster).
 
 - Clusters tagged as `test`, `stage`, or `prod` do not auto-pause.
 - Paused clusters are automatically deleted after 30 consecutive paused days. You can change the tag to avoid cluster deletion.
 - No data is lost while a cluster is paused. All execution and configuration is saved, but cluster components such as Zeebe and Operate are temporarily disabled until you resume the cluster.
+- When a cluster is paused, not all functionality is limited. For example, you may still execute BPMN timers and BPMN message catch events.
```
Comment (mesellings) [inline versioned_docs/version-8.5/components/concepts/clusters.md:46]: See previous rewording suggestion

**Review by mesellings — CHANGES_REQUESTED** [review]
Just a rewording suggestion, but I think we need to remove the auto-pause caution now as it is no longer relevant?

**Comment by christinaausley** [comment]
@mesellings Given the changes over the past few weeks with auto-pause, I'm just going to close this out 👍 Thank you for your review!

**Comment by mesellings** [comment]
Sounds good thank you! 👍

**Pull request closed without merging**